src/components/shared/Accordion.css

```diff
@@ -75,6 +75,7 @@
 .accordion ._content {
   border-bottom: 1px solid var(--theme-splitter-color);
   font-size: var(--theme-body-font-size);
+  overflow: auto;
```

Comment:
It seems all the other panels are handling overflow themselves. Maybe we should do that for Scopes too, instead of handling it in Accordion?

Reply:
@fvsch do you mean adding this `overflow: auto;` in each of the panels? I added in accordion because it seemed a more general bug affecting not only the Scopes panel only, as you can see in the image.
I will change it anyway and see how it looks like.
![Screen Shot 2019-03-24 at 22 30 35 (2)](https://user-images.githubusercontent.com/881000/54881693-f1bf3380-4e84-11e9-9a45-f6dda8c7c39a.png)

Reply:
Done! :)

Reply:
My concern is that since we already have `overflow: auto` defined for a lot of SecondaryPanes content, generally with this structure:

```html
<li>
  <div class="_header">...</div>
  <div class="_content">
    <div class="some-container-class">...</div<
  </div>
</li>
```

...where "some-container-class" already has `overflow: auto`, if we add this style to all the `_content` classes than we have two parent/child containers with this style. I'd like to avoid that. For the time being, can you avoid declaring this overflow style for all `.accordion ._content` elements, and only for the panes that are missing this style?